app/src/test/java/com/fstyle/structure_android/screen/main/MainPresenterTest.java

```diff
@@ -0,0 +1,77 @@
+package com.fstyle.structure_android.screen.main;
+
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.model.UsersList;
+import com.fstyle.structure_android.data.source.UserDataSource;
+import com.fstyle.structure_android.data.source.UserRepository;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
```

Comment:
ERROR: Wrong order for org.junit.Before import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports